adam/perception/high_level_semantics_situation_to_developmental_primitive_perception.py

```diff
@@ -585,7 +586,8 @@ def _perceive_property_assertions(self) -> None:
                     for r in self._situation.always_relations
                 )
             ):
-                properties_to_perceive.append(TWO_DIMENSIONAL)
+                # properties_to_perceive.append(TWO_DIMENSIONAL)
+                pass
```

Comment:
Why not add this property?

Reply:
@denizbeser and I ended up having the problem that the subset object learner couldn't recognize juice in a cup because it had only observed it "free-standing," which was causing issues down the line with the semantics experiment. For debugging purposes (?) we simply disabled adding the property.

Reply:
Hmm, it seems like the better solution here is to show the learner a liquid in a container in the training stages rather than removing this property?

Reply:
I think I agree.

Reply:
I couldn't find an existing curriculum for that. I suggest we leave it as is given the time constraints.

Reply:
Ok, @denizbeser  let's open a git issue to track this change then so it is explained why this occurred and link to it in the code.